create_books.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE books_db;
```

Comment:
**Observations:** The SQL script creates a database and a table with basic fields. However, it lacks constraints and indexes that could enhance performance and data integrity.

**Recommendations:**
- **Add NOT NULL Constraints:**
  - Ensure that fields like `title`, `author`, and `year` are not null to maintain data integrity.
  - Example:
    ```sql
    CREATE TABLE books (
        id INT AUTO_INCREMENT PRIMARY KEY,
        title VARCHAR(255) NOT NULL,
        author VARCHAR(255) NOT NULL,
        year INT NOT NULL
    );
    ```

- **Consider Indexes:**
  - Add indexes on columns that will be frequently queried, such as `title` and `author`, to improve query performance.